site/src/pages/index.js

```diff
@@ -88,8 +94,7 @@ function AboutProductPane() {
         <div className="shot-casing">
         <div className="shot"></div>
         </div>
-        <BubbleLayer travel={40}>
-          <Bubble size={30} thickness={3} duration={6} left="52%" bottom="0" />
+        <BubbleLayer>
```

Comment:
@romario-agc - I think we can just get rid of this BubbleLayer now, since it doesn't have any bubbles in it, right?

Reply:
Sure.